lite/base_certifier.go

```diff
@@ -0,0 +1,72 @@
+package lite
+
+import (
+	"bytes"
+
+	lerr "github.com/tendermint/tendermint/lite/errors"
+	"github.com/tendermint/tendermint/types"
+	cmn "github.com/tendermint/tendermint/libs/common"
+)
+
+var _ Certifier = (*BaseCertifier)(nil)
```

Comment:
Maybe add short comment what this line does. Unless this is very idiomatic way of ensuring static type checks.

Reply:
It's pretty idiomatic (for us, it's everywhere).